src/main/resources/application.properties

```diff
@@ -10,3 +10,4 @@ logging.level.org.hibernate.SQL=debug
 spring.jpa.show-sql=true
 spring.jpa.generate-ddl=true
 
+DEBUG=true
```

Comment:
This should probably be placed in `localhost.properties` or not committed at all. 
This will clutter the logs on the production version of the app.